doc/gui/extension/example_library/front-end/src/ListItem.tsx

```diff
@@ -0,0 +1,50 @@
+import React, { useMemo } from "react";
+import { LoV, useLovListMemo } from "taipy-gui";
+
+interface ToDoListProps {
+    lov?: LoV;
+    defaultLov?: string;
+    sort?: "asc" | "desc";
+}
+
+const styles = {
+    listItem: {
+        display: "flex",
+        alignItems: "center",
+    },
+    listItemImage: {
+        marginRight: "8px",
+        width: "1em",
+        height: "1em",
+    },
+};
+
+const ListItem: React.FC<ToDoListProps> = ({ lov, defaultLov = "", sort }) => {
```

Comment:
this is not a listItem as it shows full list ?

Reply:
Name changed